drivers/hmis/app/models/hmis/form/definition_validator.rb

```diff
@@ -11,7 +11,9 @@ def self.perform(...)
 
   # @param [Hash] document is a form definition document {'item' => [{...}] }
   # @param [role] role of the form as string ('INTAKE', 'CLIENT', etc). If not provided, HUD rule validation will not occur.
-  def perform(document, role = nil)
+  # @param [owner_type] owner_type of the form, such as Hmis::Hud::CustomAssessment
+  # @param [boolean] is_publishing if true, validate CDEDs
+  def perform(document, role = nil, owner_type = nil, is_publishing = false)
```

Comment:
And can we adjust [this code](https://github.com/greenriver/hmis-warehouse/blob/26ef8c27b51dba80b065dfa9a3034722f165a15e/drivers/hmis/app/graphql/types/hmis_schema/query_type.rb#L243) as well to skip the cded check? This runs from the JSON Editor so we don't want that to fail when adding a new field to a draft.